mk_verificator/tests/system/mirgation/test_mirgation.py

```diff
@@ -0,0 +1,66 @@
+import novaclient.client as client
+import mk_verificator.utils as utils
+
+import time
+
+
+def wait_for_vm_status_is_active(client, vm_id, retry_delay=5, timeout=60):
```

Comment:
let's move it to the separated god-class OpenStackClient(). We may discuss this approach next Monday.